avocado/utils/disk.py

```diff
@@ -87,3 +94,41 @@ def get_filesystem_type(mount_point='/'):
             _, fs_file, fs_vfstype, _, _, _ = mount_line.split()
             if fs_file == mount_point:
                 return fs_vfstype
+
+
+def get_io_scheduler_list(device_name):
+    """
+    Returns io scheduler available for the IO Device
+    :param device_name: Device  name example like sda , hda
+    :return: list of IO scheduler
+    """
+    names = open(__sched_path(device_name)).read()
```

Comment:
You are using context manager all over the code, so just for the sake of consistency, my suggestion is to use a context manager here too.

Reply:
sure make sense